.travis.yml

```diff
@@ -12,6 +12,7 @@ matrix:
 install: pip install tox
 
 script:
+  - python setup.py build
```

Comment:
Put the build commands in the `tox.ini`, example: https://opendev.org/opendev/bindep/src/branch/master/tox.ini#L13-L14

Reply:
This will be replaced by GitHub actions.

Reply:
I've removed this file.